File: week-02/day-01/draw_square.py
# Write a program that reads a number from the standard input, then draws a
# square like this:
#
#
# %%%%%
# %   %
# %   %
# %   %
# %   %
# %%%%%
#
# The square should have as many lines as the number was


a = 10
char = '%'
empty = ' '
for i in range(a):
    if i == 0 or i == a - 1:
        print( (a-1) * char)
    else:
        print( char + (a - 3) * empty + char)

File: week-07/day-01/draw-square.js
'use strict';

var lineCount = 9;

// Write a program that draws a
// square like this:
//
//
// %%%%%
// %   %
// %   %
// %   %
// %   %
// %%%%%
//
// The square should have as many lines as lineCount is

let sign = "%";
let empty = " ";

console.log(sign.repeat(lineCount - 1));
for (let i = 0; i < lineCount - 2; i++) { 
    console.log(sign + empty.repeat(lineCount - 3) + sign);
}
console.log(sign.repeat(lineCount - 1));